jovian/utils/commit.py

```diff
@@ -248,11 +249,22 @@ def _attach_file(path, gist_slug, version, output=False):
         log(str(e) + " (" + path + ")", error=True)
 
 
-def _attach_files(paths, gist_slug, version, output=False):
+def _attach_files(paths, gist_slug, version, output=False, exclude=None):
     """Helper functions to attach files & folders to a commit"""
-    # Skip if empty
+    # Look for config if empty
     if not paths or len(paths) == 0:
-        return
+        creds = read_creds()        
+        try:
+            upload_wd = creds['DEFAULT_CONFIG']['UPLOAD_WORKING_DIRECTORY']
+            if not upload_wd or output:
+                return
+        except KeyError:
+            # config not set
+            return
+
+        paths = glob.glob('**/*', recursive=True)
```

Comment:
Do we need `glob` here? The code below already performs recursive upload, right?

Reply:
I guess we could change it to just `os.listdir()` then?

In the code below, first branch gets skipped in each iteration when we use `glob`.
https://github.com/JovianML/jovian-py/blob/961a444ddcf822082957bbb587a604bc4aea3428/jovian/utils/commit.py#L263-L272